app/lib/view/settings/subsettings/supported_features.dart

```diff
@@ -12,15 +12,6 @@ class SupportedFeaturesOverviewScreen extends StatefulWidget {
 }
 
 
-final List<SPHAppEnum> supportedApps = [
-  SPHAppEnum.nachrichten,
-  SPHAppEnum.vertretungsplan,
-  SPHAppEnum.meinUnterricht,
-  SPHAppEnum.kalender,
-  // SPHAppEnum.logout.php,  // relevant?
```

Comment:
Der gedanke war, dass es in dem Unterstützungs-Tab als unterstützt angezeigt wird. Aber so ist das auch valide und nachvollziehbar.